hydra-node/src/Hydra/HeadLogic.hs

```diff
@@ -586,12 +883,27 @@ onOpenNetworkAckSn Environment{party} openState otherParty snapshotSignature sn
 --
 -- __Transition__: 'OpenState' → 'OpenState'
 onOpenClientClose ::
+  Monoid (UTxOType tx) =>
   OpenState tx ->
   Outcome tx
 onOpenClientClose st =
-  cause OnChainEffect{postChainTx = CloseTx headId parameters confirmedSnapshot}
+  -- Spec: missing?
+  --       η ← combine(S⁻.𝑈)
+  --       ηω ← combine(outputs(S⁻.txω))
+  --       ξ ← S⁻.σ
+  --       postTx (close, S⁻.v, S⁻.s, η, ηω,ξ)
```

Comment:
We basically pass the `snObj` to close tx construction. However, we should update the code and spec to pick the right `CloseType` (or `CloseAction` case) here.